src/data/effect_commands.asm

```diff
@@ -238,67 +238,67 @@ ParasectSporeEffectCommands:
 
 WeedlePoisonStingEffectCommands:
 	dbw EFFECTCMDTYPE_BEFORE_DAMAGE, Poison50PercentEffect
-	dbw EFFECTCMDTYPE_AI, $4b27
+	dbw EFFECTCMDTYPE_AI, WeedlePoisonSting_AIEffect
 	db  $00
 
 IvysaurPoisonPowderEffectCommands:
 	dbw EFFECTCMDTYPE_BEFORE_DAMAGE, PoisonEffect
-	dbw EFFECTCMDTYPE_AI, $4b2f
+	dbw EFFECTCMDTYPE_AI, IvysaurPoisonPowder_AIEffect
 	db  $00
 
 BulbasaurLeechSeedEffectCommands:
-	dbw EFFECTCMDTYPE_AFTER_DAMAGE, $4b37
+	dbw EFFECTCMDTYPE_AFTER_DAMAGE, BulbasaurLeechSeedEffect
 	db  $00
 
 VenusaurEnergyTransEffectCommands:
-	dbw EFFECTCMDTYPE_INITIAL_EFFECT_2, $4b44
-	dbw EFFECTCMDTYPE_BEFORE_DAMAGE, $4b77
-	dbw EFFECTCMDTYPE_AFTER_DAMAGE, $4bfb
-	dbw EFFECTCMDTYPE_REQUIRE_SELECTION, $4b6f
+	dbw EFFECTCMDTYPE_INITIAL_EFFECT_2, EnergyTrans_CheckPlayArea
+	dbw EFFECTCMDTYPE_BEFORE_DAMAGE, EnergyTrans_TransferEffect
+	dbw EFFECTCMDTYPE_AFTER_DAMAGE, EnergyTrans_AIEffect
```

Comment:
Seems like AIEffect may be wrong here? Don't know what the most appropriate name would be, maybe some follow up to TransferEffect.